api/src/main/scala/uk/ac/wellcome/platform/api/models/Record.scala

```diff
@@ -3,13 +3,17 @@ package uk.ac.wellcome.platform.api.models
 import com.fasterxml.jackson.annotation.JsonProperty
 import com.sksamuel.elastic4s.searches.RichSearchHit
 import org.elasticsearch.action.get.GetResponse
-import uk.ac.wellcome.models.{IdentifiedWork, Work}
+import uk.ac.wellcome.models.{Agent, IdentifiedWork, Period, Work}
 import uk.ac.wellcome.utils.JsonUtil
 
 case class Record(
   @JsonProperty("type") ontologyType: String = "Work",
   id: String,
-  label: String
+  label: String,
+  description: Option[String] = None,
+  lettering: Option[String] = None,
+  hasCreatedDate: Option[Period] = None,
+  hasCreator: List[Agent] = List()
```

Comment:
I didn't get a chance to review this yesterday, but is it a problem that this seems to track the `Work` type so closely?